src/main/java/org/skife/jdbi/v2/Query.java

```diff
@@ -420,4 +426,13 @@ public void registerMapper(ResultSetMapperFactory m)
     {
         this.mappingRegistry.add(m);
     }
+
+    public void registerColumnMapper(ResultColumnMapper m)
```

Comment:
I know this is just mirroring existing design decisions, but I think adding methods on `Query` which modify state scoped more appropriately to the `Handle` or even `DBI` itself is bad.  Without understanding the internals, I might assume this register method only applies to the scope of the `Query`.  Maybe we break with tradition here and omit these convenience methods?